### ISSUE TITLE
Remote captive core fails to start

### ISSUE THREAD

**Issue by 2opremio** [stellar/go#3230]
While trying to retest #3228 with master ( b361462 ) I noticed that prepare-range doesn't work anymore.

I noticed it when using the remote captive-core http server.

After some binary testing, I have found that the culprit is at e7cf49028ba

To reproduce, you can run https://github.com/stellar/kube/tree/master/local/horizon-captive-core with the following patch (although I guess it also affects local captive-core):

```diff
diff --git a/local/horizon-captive-core/kustomization.yaml b/local/horizon-captive-core/kustomization.yaml
index cd2acd74..7c451c41 100644
--- a/local/horizon-captive-core/kustomization.yaml
+++ b/local/horizon-captive-core/kustomization.yaml
@@ -37,9 +37,11 @@ patchesJson6902:
       kind: Deployment
       name: horizon-with-remote-core
 # Uncomment to use a custom (myimage:mytag) horizon container image (e.g. built locally)
-# images:
-#  - name: stellar/horizon
-#    newName: myimage
-#    newTag: mytag
+images:
+ - name: stellar/horizon
+   newTag:   e7cf49028
 # For convenience, create all the resources in the default namespace
 namespace: default
```

**Comment by bartekn** [comment]
@2opremio possibly fixed in #3231.

**Comment by 2opremio** [comment]
I can confirm it's fixed
